### ISSUE TITLE
Accessing Bluetooth Serial Profile

### ISSUE THREAD

**Issue by mumairofficial** [pauldemarco/flutter_blue#88]
This is not the issue I need help; will this plugin help me to connected with bluetooth devices with serial porfile such as arduino etc.

**Comment by brianalois** [comment]
Yes how do we scan for devices that are not low energy, and how would we access the serial port?

**Comment by bigship** [comment]
For iOS platform, flutter_blue use the native CoreBluetooth framework under the hood, and CoreBluetooth only targets BlueTooth Low Energy. So I don't think it's possible.

**Comment by edufolly** [comment]
Hi folks, I tried to find a plugin to do the interface with an HC-05 Bluetooth adapter, but I didn't find. So I started to dev one, based on flutter-blue and I would like to get your opinions and improvements. I'm waiting for your comments. https://github.com/edufolly/flutter_bluetooth_serial

**Comment by stancecoke** [comment]
My HM10 Module offers the service 0000FFE0-0000-1000-8000-00805F9B34FB. This has read and write properties to receive and send the UART messages.
